ccd/templates/admin-web.yaml

```
@@ -0,0 +1,115 @@
+{{- if .Values.adminWeb.enabled }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ .Release.Name }}-admin-web
+  labels:
+    app.kubernetes.io/name: {{ .Release.Name }}-admin-web
+    helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}-admin-web
+    app.kubernetes.io/instance: {{ .Release.Name }}-admin-web
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: {{ .Release.Name }}-admin-web
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: {{ .Release.Name }}-admin-web
+    spec:
+      containers:
+      - image: {{ .Values.adminWeb.image }}
+        name: {{ .Release.Name }}-admin-web
+        env:
+        - name: IDAM_OAUTH2_CLIENT_ID
+          value: ccd_admin
+        - name: HPKP_MAX_AGE
+          value: 2592000
```

can you quote this for safety?
```suggestion
          value: '2592000'
```